src/main/java/org/entur/gbfs/mapper/SystemInformationAdditionalMapper.java

```diff
@@ -16,6 +16,7 @@
 
 @Mapper(imports = {List.class})
 public abstract class SystemInformationAdditionalMapper {
+    public static final String DEFAULT_OPENING_HOURS = "Mo-Su,PH 00:00-24:00";
```

Comment:
I support adding "Always open" as default opening hours. I wonder if `24/7` would not do the same job and be more common(?) 

The OpeningHours Evaluation tool does not show any warnings or hints suggesting that [24/7](https://openingh.openstreetmap.de/evaluation_tool/?EXP=24%2F7&lat=48.71677551638915&lon=9.09648801869233&mode=0&DATE=1758005760000) would not also work for public holidays.